docs/config.md

```diff
@@ -207,6 +207,27 @@ The partner registry requires authentication for pulling images.  This can be ac
 docker --config=/var/lib/origin/.docker login -u <registry-user> -p <registry-password> registry.connect.redhat.com
 ```
 
+### API V2 Registry
+
+Registries that implements the Docker [Registry HTTP API V2](https://docs.docker.com/registry/spec/api/) protocol can be configured with the `apiv2` registry adapter as follows
+
+```yaml
+registry:
+  - name: jfrog
```

Comment:
Rather than suggest a specific name, I'd make this a pseudo-template like the values below. That would make it clear that the user can supply their own value.